alphafold3_pytorch/utils/data_utils.py

```diff
@@ -1,6 +1,8 @@
-from typing import Set
+from typing import Any, Dict, Literal, Set
 
-from alphafold3_pytorch.tensor_typing import ChainType, ResidueType, typecheck
+import numpy as np
+
+from alphafold3_pytorch.utils.tensor_typing import ChainType, ResidueType, typecheck
```

Comment:
think it should be `from alphafold3_pytorch.tensor_typing`

Reply:
Good catch. Fixed